addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1125,6 +1128,14 @@ def _load_more_members(self, known_member_ids):
         )
         return Store(unknown_members).add(self, {"memberCount": count}).get_result()
 
+    def _get_channel_history(self):
+        """
+        Converting message body back to plaintext for correct data formatting in HTML field.
+        """
+        return Markup('').join(
+            Markup('%s: %s<br/>') % (message.author_id.name, html2plaintext(message.body))
+            for message in self.message_ids.sorted('id')
+        )
```

Comment:
It's not a great method, we don't need that at all in base discuss code.

It's even more strange if you keep the duplicate method in `im_livechat`, because then this one could just be in `crm`.

Reply:
@seb-odoo PR is done now, you can review it again if you want. only thing I didn't improve was this function, because I don't think it's possible to separate `livechat` and `crm` without having those two duplicate functions (well they are almost duplicate not 100% :) ) for me personally since function is small it's not a big deal. I will do as you tell me, I just couldnt' find any other way

Reply:
Ok to keep it in `mail` to not add a bridge module, but the method should not be fully duplicated in livechat to add a single `or` in the middle.

Reply:
So make a separate method to get the name from the message and override only that part.